src/lib/isChildPath.js

```diff
@@ -0,0 +1,22 @@
+export default function isChildPath (parent, child) {
+  if (parent === "/") {
+    return true;
+  }
+
+  const parentSplit = parent.split("/");
+  const childSplit = child.split("/");
+  const parentSplitLength = parentSplit.length;
+
+  if (parentSplitLength > childSplit.length) {
+    return false;
+  }
+
+  for (let i = 0; i < parentSplitLength; i++) {
+    if (parentSplit[i] !== childSplit[i]) {
+      return false;
+    }
+  }
+
+  return true;
+}
```

Comment:
Another thing we can do here is use `path.relative(parent, child)` and if the path starts with `.` then we know it isn't a parent.